spring-pulsar-spring-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/PulsarProperties.java

```diff
@@ -1323,6 +1323,21 @@ public static class Listener {
 		 */
 		private SchemaType schemaType;
 
+		/**
+		 * Max number of messages for batch.
+		 */
+		private int maxNumMessages = -1;
+
+		/**
+		 * Max number of bytes for batch.
+		 */
+		private int maxNumBytes = 10 * 1024 * 1024;
+
+		/**
+		 * Timeout for batch.
```

Comment:
It may be helpful to  elaborate on "for batch" (this applies to all 3 of the new fields)